### DISCUSSION TITLE
About the release

### DISCUSSION THREAD

**Discussion by Younguz** [giswqs/geohey#9] [Q&A] [answered]
Sorry to bother you, when I finished the sync my file to github first time, all my file disappeared. And after I redo that, when I trying to release the update, it show the error, like that: ERROR    HTTPError: 400 Bad Request from https://upload.pypi.org/legacy/        
         File already exists. See https://pypi.org/help/#file-name-reuse for    
         more information.                                                      
Error: Process completed with exit code 1.

**Comment by giswqs** [comment, marked as answer]
Run `bump-my-version bump minor` to increase the version number to `0.1.0` so that it does not conflict with the existing versions on PyPI.

**Reply by Younguz** [reply]
sorry where should I run this code? On visual studio?

**Reply by giswqs** [reply]
Check [today's lecture recording](https://youtu.be/9BSup9uMl1E?si=EAh9Hlt5d_z7WYNQ&t=2405).
